### ISSUE TITLE
Convert video coder？

### ISSUE THREAD

**Issue by Haibo-Zhou** [lisamelton/video_transcoding#347]
Can it convert for instance `Avatar.The.Way.of.Water.2022.BluRay.1080p.x265.10bit.HEVC.2Audio.AAC.6CH.NORMTEAM.FZ.mkv` this x265.10bit to x264 encode format? 

The context is my TV cannot support decoding x265 and 10 bit, thus I suppose to convert it first.

**Comment by lisamelton** [comment]
@Haibo-Zhou Yes, but it won't handle the conversion of HDR to SDR if the input is in HDR format. You might be better off trying my newer transcoding project:

https://github.com/lisamelton/other_video_transcoding

**Comment by Haibo-Zhou** [comment]
@lisamelton Thanks, could you provide a example to achieve above conversion? Like in Mac terminal what command and parameter should be used.

**Comment by stephenix** [comment]
unsubscribe
















At 2024-04-17 16:12:07, "Zhou Haibo" ***@***.***> wrote:

@lisamelton Thanks, could you provide a example to achieve above conversion? Like in Mac terminal what command and parameter should be used.

**Comment by lisamelton** [comment]
@Haibo-Zhou My apologies. I missed the notification for your comment.

You don't need any special options for a HDR to SDR conversion. The `other-transcode` will default to H.264 output so that should just work. If it doesn't, try forcing the software encoder by using the `--x264-cbr` option.

**Comment by Haibo-Zhou** [comment]
> @Haibo-Zhou My apologies. I missed the notification for your comment.
> 
> You don't need any special options for a HDR to SDR conversion. The `other-transcode` will default to H.264 output so that should just work. If it doesn't, try forcing the software encoder by using the `--x264-cbr` option.

Thanks, lisamelton :)

**Comment by lisamelton** [comment]
@Haibo-Zhou The new version of `transcode-video.rb` should do this automatically now. If not, you can add `--extra colorspace=bt709` to you command line.
